mephisto/client/review/review_server.py

```diff
@@ -41,16 +42,39 @@ def json_reader(f):
         for jsonline in f:
             yield json.loads(jsonline)
 
+    def mephistoDBReader():
+        if database is None:
+            return []
```

Comment:
Perhaps we should print an error message or something here instead to denote that no task_name in the database was specified

Reply:
this also prob never gets run because Line 71 below only gets run if Line 70 is true and database is not None

Reply:
How about if no task_name is specified, we print out an error message, as well as a list of the current task_names that are stored in the db (perhaps sorted by desc updated date)? I think this should be a huge UX win

Reply:
This is actually intentional as a safety mechanism for future use of the method however I can remove it as I realize it is a little messy. There is also no way to distinguish between the db flag being given no argument or the db flag not being called so I didn't include an error message